internal/observability.go

```diff
@@ -65,42 +67,111 @@ var ViewExportedSpans = &view.View{
 	TagKeys:     []tag.Key{tagKeyExporterName},
 }
 
+var mLatencyMs = stats.Float64("oc.io/latency", "The latency of the various actions in milliseconds", "ms")
+
+var latencyDistribution = view.Distribution(
+	0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80,
```

Comment:
less than 0 is never going to be used, right? at least that is the way that this ends up in Prometheus.